scripts/ci/main.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+"""Testing NV12 image encoding."""
```

Comment:
In `scripts/ci/main.py`? O.o Is this really the main CI check?

Reply:
wait what. what happened here Oo. This file slipped into ci/main for no reason at all. It's the same in tests/python/release_checklist
I have no idea how this came to be and why I didn't notice on review. Maybe it has something to do with the odd .pyd file I leaked in